src/main.cpp

```diff
@@ -137,6 +161,15 @@ main(int argc, char *argv[])
         parser.addVersionOption();
         QCommandLineOption debugOption("debug", "Enable debug output");
         parser.addOption(debugOption);
+
+        // This option is not actually parsed via Qt due to the need to parse it before the app
+        // name is set. It only exists to keep Qt from complaining about the --profile/-p
+        // option and thereby crashing the app.
+        QCommandLineOption configName(QStringList() << "p" << "profile",
+                "Create a unique configuration file", 
+                "config", "config name");
```

Comment:
```suggestion
                tr("Create a unique profile, which allows you to log into several accounts at the same time and start multiple instances of nheko."), 
                tr("profile"), tr("profile name"));
```

Maybe tr doesn't work though, and you will need to use QCoreApplication::translate instead.